apps/files_sharing/lib/Controller/ShareAPIController.php

```
@@ -443,6 +447,16 @@ public function createShare(
 					\OCP\Constants::PERMISSION_DELETE);
 			}
 			$share->setSharedWith($shareWith);
+		} else if ($shareType === \OCP\Share::SHARE_TYPE_CIRCLE) {
+
+			$circle = \OCA\Circles\Api\Circles::detailsCircle($shareWith);
```

This will fail hard if the app is not installed... Of course then there probabaly won't be a circle share. But if the app gets disabled or so. It will most likely do :boom: